internal/npm_install/npm_install.bzl

```diff
@@ -26,6 +26,11 @@ load("//internal/common:os_name.bzl", "is_windows_os")
 load("//internal/node:node_labels.bzl", "get_node_label", "get_npm_label", "get_yarn_label")
 
 COMMON_ATTRIBUTES = dict(dict(), **{
+    "timeout": attr.int(
+        default = 3600,
+        doc = """Maximum duration of the command "yarn install" in seconds
+        (default is 3600 seconds).""",
```

Comment:
nit: shouldn't need to write the default here, probably we'll find it appears twice in the generated doc

Reply:
👍